Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby
+MAINTAINER Jonathan Claudius
+ENV PROJECT=github.com/mozilla/ssh_scan
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
```

Comment:
I'm wondering if adding this on line 9 would do the trick...

rvm --create --ruby-version use ruby-$(ruby -e 'puts RUBY_VERSION')@ssh_scan